pkgs/tools/compression/bzip2/default.nix

```diff
@@ -44,6 +44,10 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  postInstall = ''
+    ln -s libbz2.so.1.0.* $out/lib/libbz2.so.1.0
```

Comment:
@vcunat is right. This should be more correct (but please test :)):

```suggestion
    ln -s $out/lib/libbz2.so.1.0.* $out/lib/libbz2.so.1.0   
```

Reply:
I started my own build just now.

Reply:
Ah, I was thinking about
```bash
(cd "$out/lib"; ln -s libbz2.so.1.0.* libbz2.so.1.0)
```

Reply:
Nix will normalizes symlinks in the end so both variants end up in the same build output.

Reply:
I didn't know that, but I wouldn't mind even an absolute symlink.